frontend/src/App.scss

```diff
@@ -8,6 +8,16 @@
   src: local('Suisse Intl Book'), url(./fonts/SuisseIntl-Book.woff) format('woff');
 }
 
+@font-face {
+  font-family: 'Messer Condensed';
+  src: local('Messer Condensed'), url(./fonts/MesserV2.1TRIAL-Condensed.woff) format('woff');
+}
+
+@font-face {
+  font-family: 'Suisse Intl Book';
+  src: local('Suisse Intl Book'), url(./fonts/SuisseIntl-Book.ttf) format('ttf');
```

Comment:
The files that are referenced here are also added in this PR. They should removed from the PR before merging.

Reply:
Fixing in a new commit in this PR.